content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,95 @@ title: Custom Metrics
+You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][4].
 
-## Sending custom metrics from the Datadog Lambda library
+## Enabling synchronous custom metrics
 
-{{< img src="serverless/serverless_custom_metrics.png" alt="Collecting Custom Metrics from AWS Lambda" >}}
+If you want to generate custom metrics from a Lambda function that is written in Python, Node.js or Golang and is performance-critical, you can use Datadog Lambda Library to submit custom metrics synchronously to Datadog API at the end your Lambda invocation.
 
-Install the Datadog Lambda Library to collect and send custom metrics. Metrics sent from the Datadog Lambda Library are automatically aggregated into [distributions][3], so you can graph the `avg`, `sum`, `max`, `min`, and `count`. You can also calculate aggregations over a set of tags for the 50th, 75th, 95th, and 99th percentile values on the [Distribution Metrics][3] page.
+1. Follow the general [serverless installation instructions][5] to install the Datadog Lambda Library and configure your Lambda function. Note, you do _NOT_ need to add the Datadog Lambda Extension or Forwarder.
+1. Ensure the environment variable `DD_FLUSH_TO_LOG` is set to `false`, which instructs the Datadog Lambda Library to send metrics synchronously to the Datadog API.
+1. Ensure the environment variable `DD_TRACE_ENABLED` is set to `false`, which instructs the Datadog Lambda Library to NOT generate traces for the Lambda function.
+1. Ensure the environment variable `DD_API_KEY` (or an equivalent one when encrypted) is set with a valid Datadog API key.
+1. Import and use the helper function from the Datadog Lambda Library, such as `lambda_metric` or `sendDistributionMetric`, to submit your custom metrics following the [sample code](#custom-metrics-sample-code).
+### Running in a VPC
 
-Distribution metrics are designed to instrument logical objects, like services, independent of the underlying hosts. So, they are well-suited for serverless infrastructure because they aggregate metrics server-side instead of locally with an Agent.
+The Datadog Lambda Library requires [access to the public internet][6] to submit custom metrics **synchronously**. If your Lambda function is associated with a VPC, ensure that your function can reach the public internet.
 
-### Synchronous vs. asynchronous custom metrics
+## Enabling asynchronous custom metrics
 
-The Datadog Lambda Library supports submitting custom metrics in Lambda, both synchronously and asynchronously.
+If you want to generate custom metrics from a performance-critical Lambda function, Datadog encourage you to use Datadog Lambda Library to submit custom metrics asynchronously. The Datadog Lambda Library can either
```

Comment:
```suggestion
If you want to generate custom metrics from a performance-critical Lambda function, Datadog encourages you to use Datadog Lambda Library to submit custom metrics asynchronously. The Datadog Lambda Library can either
```

Reply:
It looks like this sentence is unfinished

Reply:
OK, I think what you should change this as follows:

The Datadog Lambda Library can either**:** <-- add colon
- Publishes -> publish...
- Emits -> emit...